releases/Dockerfile.linux

```diff
@@ -0,0 +1,31 @@
+FROM alpine:3.6
+
+ARG ACSENGINE_VERSION
+ARG BUILD_DATE
+
+# Metadata as defined at http://label-schema.org
+LABEL maintainer="Microsoft" \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.vendor="Microsoft" \
+      org.label-schema.name="Azure Container Service Engine (acs-engine)" \
+      org.label-schema.version=$ACSENGINE_VERSION \
+      org.label-schema.license="MIT" \
+      org.label-schema.description="The Azure Container Service Engine (acs-engine) generates ARM (Azure Resource Manager) templates for Docker enabled clusters on Microsoft Azure with your choice of DCOS, Kubernetes, or Swarm orchestrators." \
+      org.label-schema.url="https://github.com/Azure/acs-engine" \
+      org.label-schema.usage="https://github.com/Azure/acs-engine/blob/master/docs/acsengine.md" \
+      org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.vcs-url="https://github.com/Azure/acs-engine.git" \
+      org.label-schema.docker.cmd="docker run -v \${PWD}:/acs-engine/workspace -it --rm microsoft/acs-engine:$ACSENGINE_VERSION"
+
+RUN apk add --update bash curl && \
+    rm -rf /var/cache/apk/* 
+     
+RUN curl -L "https://github.com/Azure/acs-engine/releases/download/v${ACSENGINE_VERSION}/acs-engine-v${ACSENGINE_VERSION}-linux-amd64.tar.gz" | tar xvz -C ~ && \

```

Comment:
Instead of downloading the binary from the internet like this we could just supply it as part of the build step. This way we can release the binary and docker image at the same time.

Reply:
I'm more than happy for someone who undestands the release pipeline details to make the update to the Dockerfile.